### PR TITLE
Fix embarrassing query error in mz_cluster_replica_utilization

### PR DESCRIPTION
**Pull request by umanwizard**
I somehow didn't realize `/` does integer division, so these will always be zero.

### CODE REVIEW

File: src/adapter/src/catalog/builtin.rs [hunk 1]
```diff
@@ -2265,8 +2265,8 @@ pub const MZ_CLUSTER_REPLICA_UTILIZATION: BuiltinView = BuiltinView {
 SELECT
     r.id AS replica_id,
     m.process_id,
-    m.cpu_nano_cores / s.cpu_nano_cores * 100 AS cpu_percent,
-    m.memory_bytes / s.memory_bytes * 100 AS memory_percent
+    100 * m.cpu_nano_cores / s.cpu_nano_cores AS cpu_percent,
+    100 * m.memory_bytes / s.memory_bytes AS memory_percent
```
Comment (umanwizard) [inline src/adapter/src/catalog/builtin.rs:2269]:
```suggestion
    m.cpu_nano_cores::float8 / s.cpu_nano_cores AS cpu_percent,
    m.memory_bytes::float8 / s.memory_bytes AS memory_percent
```

File: src/adapter/src/catalog/builtin.rs [hunk 2]
```diff
@@ -2265,8 +2265,8 @@ pub const MZ_CLUSTER_REPLICA_UTILIZATION: BuiltinView = BuiltinView {
 SELECT
     r.id AS replica_id,
     m.process_id,
-    m.cpu_nano_cores / s.cpu_nano_cores * 100 AS cpu_percent,
-    m.memory_bytes / s.memory_bytes * 100 AS memory_percent
+    m.cpu_nano_cores::float8 / s.cpu_nano_cores AS cpu_percent,
+    m.memory_bytes::float8 / s.memory_bytes AS memory_percent
```
Comment (umanwizard) [inline src/adapter/src/catalog/builtin.rs:2269]:
```suggestion
    m.cpu_nano_cores::float8 / s.cpu_nano_cores * 100 AS cpu_percent,
    m.memory_bytes::float8 / s.memory_bytes * 100 AS memory_percent
```

**Review by jkosh44 — APPROVED** [review]
LGTM, though we should add a test that just checks that these columns are greater than 0.

**Comment by umanwizard** [comment]
Done

**Pull request merged**